tests/ImageSharp.Tests/ImageSharp.Tests.csproj

```diff
@@ -2,26 +2,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
     <DebugSymbols>True</DebugSymbols>
     <AssemblyName>SixLabors.ImageSharp.Tests</AssemblyName>
     <Platforms>AnyCPU;x64;x86;ARM64</Platforms>
     <RootNamespace>SixLabors.ImageSharp.Tests</RootNamespace>
     <Configurations>Debug;Release</Configurations>
   </PropertyGroup>
 
```

Comment:
We shouldn’t be removing this, instead we should be multi targeting to include .NET 9

Reply:
Thanks. We'll need to update the YAML file `build-and-test.yml` now to ensure the preview framework is installed with the additional workflows.

You can use the version from the release/v3.1.x branch as a guide. 

https://github.com/SixLabors/ImageSharp/blob/4584377879f8e83cd351425c665a9b691047804b/.github/workflows/build-and-test.yml